server/tso/tso.go

```diff
@@ -39,6 +39,8 @@ const (
 	// When a TSO's logical time reaches this limit,
 	// the physical time will be forced to increase.
 	maxLogical = int64(1 << 18)
+	// MaxSuffixBits indicates the max number of suffix bits.
+	MaxSuffixBits = 8
```

Comment:
```suggestion
	MaxSuffixBits = 4
```

Reply:
I will update it with the `go.mod` file after merging the pingcap/kvproto#701.